elide-graphql/src/test/java/example/Book.java

```diff
@@ -51,6 +54,10 @@
     private long publishDate = 0;
     private Collection<Author> authors = new ArrayList<>();
     private Publisher publisher = null;
+    private Date publicationDate = null;
+    private Date lastPurchasedDate = null;
+    private Author.AuthorType authorType;
```

Comment:
This member doesn't make sense for this model.  Books have a collection of authors but a single authorType?

Reply:
We should update this. I was working to force the `ModelBuilder` to go through a use case where it was blowing up. We need the same enum in multiple models.